api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/audit/ClaimAuditor.java

```diff
@@ -48,4 +48,13 @@ public void auditNewCard(UUID claimId, CardResponse cardResponse) {
                 .build();
         eventLogger.logEvent(newCardEvent);
     }
+
+    public void auditMakePayment(UUID claimId, UUID paymentId, String paymentReference) {
```

Comment:
Not directly related to this PR, but I'm wondering if `ClaimAuditor` should be renamed now to something more generic like just `Auditor` or `EventAuditor`?

Reply:
That sounds reasonable probably `EventAuditor`. I'll see how many additional files it affects - maybe in a separate PR?

Reply:
Yep, a separate PR is fine.